libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c

```diff
@@ -273,6 +285,45 @@ static int32_t tlsSetup( const SocketsConfig_t * pSocketsConfig,
 
 /*-----------------------------------------------------------*/
 
+static TransportSocketStatus_t connectToServer( Socket_t tcpSocket,
+                                                const ServerInfo_t * pServerInfo )
+{
+    TransportSocketStatus_t returnStatus = TRANSPORT_SOCKET_STATUS_SUCCESS;
+    int32_t secureSocketStatus = ( int32_t ) SOCKETS_ERROR_NONE;
+    SocketsSockaddr_t serverAddress = { 0 };
+
+    /* Establish connection. */
+    serverAddress.ucSocketDomain = SOCKETS_AF_INET;
+    serverAddress.usPort = SOCKETS_htons( pServerInfo->port );
+    serverAddress.ulAddress = SOCKETS_GetHostByName( pServerInfo->pHostName );
+
+    /* Check for errors from DNS lookup. */
+    if( serverAddress.ulAddress == ( uint32_t ) 0 )
+    {
+        LogError( ( "Failed to connect to server: DNS resolution failed: hostNameLength %lu, Server=%s.",
```

Comment:
Would be fine to remove the server name length as the server name suffices
```suggestion
        LogError( ( "Failed to connect to server: DNS resolution failed: Server=%s.",
```